### ISSUE TITLE
Not translating string arrays

### ISSUE THREAD

**Issue by taimur97** [westlinkin/AndroidLocalizationer#7]
It is not translating the string arrays in my string.xml file?

**Comment by westlinkin** [comment]
Yes, it is not. This will be added as a feature in future updates

**Comment by Ludde321** [comment]
Your plugin not only ignores the string-arrays, but actually REMOVES my already translated string-arrays from output languages. 
For this reason I cannot use your plugin (yet)!

**Comment by westlinkin** [comment]
@Ludde321 I'm very sorry about that. I've been very busy with my job recently. I'll fix it as soon as possible.

**Comment by Ludde321** [comment]
Sorry if I sounded a bit negative. Your plugin is just add a few fixes away from being great!

**Comment by Photogad** [comment]
any chance you can add this? It's very annoying and your plugin is the only one out there like it.
